src/main/java/com/practicesoftwaretesting/common/asserts/NotFoundResponseAssert.java

```diff
@@ -0,0 +1,20 @@
+package com.practicesoftwaretesting.common.asserts;
+
+import com.practicesoftwaretesting.common.responses.ResponseMessage;
+
+import lombok.AllArgsConstructor;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@AllArgsConstructor
+public class NotFoundResponseAssert {
+
+    private ResponseMessage messageAfterDelete;
```

Comment:
the same, field of the `ResponseMessage` class is `message`, so why do we use `messageAfterDelete` here?